sr_robot_launch/scripts/sr_ur_release

```diff
@@ -0,0 +1,193 @@
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import rospy
+from std_msgs.msg import Bool
+from ur_dashboard_msgs.srv import GetSafetyMode, GetProgramState, GetRobotMode, Load
+from ur_dashboard_msgs.msg import SafetyMode, ProgramState, RobotMode
+from std_srvs.srv import Trigger
+
+class SrUrUnlock():
+    def __init__(self):
+        self.external_control_program_name = "external_ctrl.urp"
+        self.arms = []
+        if rospy.has_param('ra_sr_ur_robot_hw'):
+            self.arms.append('ra')
+        if rospy.has_param('la_sr_ur_robot_hw'):
+            self.arms.append('la')
+        if len(self.arms) == 0:
+            rospy.signal_shutdown("No arms detected, shutting down %s", rospy.get_name())
+        self.subscriber = rospy.Subscriber("sr_arm/release", Bool, self.pedal_sub)
+        self.rate = rospy.Rate(5)
+        while not rospy.is_shutdown():
+            try:
+                self.rate.sleep()
+            except rospy.ROSInterruptException:
+                rospy.loginfo("Shutting down %s", rospy.get_name())
+
+    def pedal_sub(self, data):
+        if data == True:
+            self.release_arm()
+
+    def call_dashboard_service(self, side, service_name):
+        service_string = "/" + side + "_sr_ur_robot_hw/dashboard/" + service_name
+        serv_call = rospy.ServiceProxy(service_string, Trigger)
+        resp = serv_call()
+
+    def call_service(self, side, service_name):
+        service_string = "/" + side + "_sr_ur_robot_hw/" + service_name
+        serv_call = rospy.ServiceProxy(service_string, Trigger)
+        resp = serv_call()
+
+    def get_safety_mode(self, arm):
+        try:
+            safety_mode_service = rospy.ServiceProxy("/" + arm + "_sr_ur_robot_hw/dashboard/get_safety_mode", GetSafetyMode)
+            safety_mode_msg = safety_mode_service()
+            return safety_mode_msg.safety_mode.mode
+        except rospy.ServiceException, e:
+            rospy.logerr("Service call to 'get_safety_mode' failed for arm %s. %s", arm, e)
+
+    def get_robot_mode(self, arm):
+        try:
+            get_mode_service = rospy.ServiceProxy("/" + arm + "_sr_ur_robot_hw/dashboard/get_robot_mode", GetRobotMode)
+            get_mode_msg = get_mode_service()
+            return get_mode_msg.robot_mode.mode
+        except rospy.ServiceException, e:
+            rospy.logerr("Service call to 'get_robot_mode' failed for arm %s. %s", arm, e)
+
+    def startup_arms(self, arms):
+        for arm in arms:
+            self.call_dashboard_service(arm, "power_on")
+        rospy.sleep(5)
+        for arm in arms:
```

Comment:
why two loops?

Reply:
It's quicker with two loops. There needs to be a 5 second delay after "power_on" and after "brake_release". With just one loop (and two arms) it would be: arm1 power_on, sleep 5, arm2 power_on, sleep 5, arm1 brake_release, sleep 5, arm2 brake_release, sleep 5. Would take twice as long